templates/zerver/emails/missed_message.txt

```diff
@@ -1,24 +1,24 @@
 Hello {{ name }},
 
-While you were away you received {{ message_count }} new message{{ messages|pluralize }}{% if mention %} in which you were mentioned{% endif %}!
-
+While you were away you received {{ message_count }} new{% if group_pm %} group{% elif private_message %} private{% endif %} message{{ message_count|pluralize }}{% if mention %} in which you were mentioned{% endif %}!
 
+{% if show_message_content %}
 {% for recipient_block in messages %}
     {{ recipient_block.header.plain }}
     {% for sender_block in recipient_block.senders %}
         {% if sender_block.sender %}{{ sender_block.sender }}{% endif %}{% for message_block in sender_block.content %}
-                {{ message_block.plain }}
```

Comment:
why this change? was it failing the linter?
This one is probably harmless, but generally just wary of spacing changes in these files, since (unilke e.g. javascript code), spaces are not ignored.

Reply:
Yes, lint tests was failing.

Reply:
yup this is odd behaviour. I will check this out soon.

Reply:
@YJDave can you tell me about that linter test a bit more. According to my theory linter shouldn't have failed on this file in any case (or passed for that matter) because linter tests are never actually run on this file! (Due to its .txt extension)
If we ran linter on this file lines like this (L8: `{% if sender_block.sender %}{{ sender_block.sender }}{% endif %}{% for message_block in sender_block.content %}`) would never pass due to `for` block starting in the same line as the previous block.

FYI: @timabbott 

(I am sorry for the delay in getting back to this.)

Reply:
@adnrs96 Sorry, for not making this more clearer before, tests were failing because of `tabs` not because of block indentation.
```
(zulip-py3-venv)vagrant@vagrant-base-trusty-amd64:/srv/zulip$ ./tools/lint
txt       | Fix tab-based whitespace at templates/zerver/emails/missed_message.txt line 10:
txt       |         	{{ message_block.plain }}
```

Reply:
I should have just replace the tabs with spaces, rather than removing indentation. Sorry for the trouble :(